mne/proj.py

```diff
@@ -391,27 +392,37 @@ def sensitivity_map(
         The forward operator.
     projs : list
         List of projection vectors.
-    ch_type : ``'grad'`` | ``'mag'`` | ``'eeg'``
+    ch_type : {'grad', 'mag', 'eeg'}
```

Comment:
revert this change. using `|` to separate parameter options is how we do it in MNE-Python (see item 3 in [this section](https://mne.tools/dev/development/contributing.html#we-mostly-follow-numpy-style-for-docstrings))